lib/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -99,6 +99,9 @@ namespace {
         //! Color array to support per-instance color and selection highlight.
         MFloatArray _instanceColors;
 
+        //! If true, associate geometric buffers to the render item and trigger consolidation/instancing update
+        bool _geometryDirty{ false };
```

Comment:
These aren't my changes, are the correct?

Reply:
Yes, come from #139